<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>y_jbylbxtdbzmzsqdjjjb</class>
 <widget class="QDockWidget" name="y_jbylbxtdbzmzsqdjjjb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基本医疗保险特定病种门诊申请登记交接表</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0,1,0">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>交接情况</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>全部</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>已上传</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>卡已收回但未领取</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>已领取</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>回退 / 作废</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>参保类别</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <item>
           <property name="text">
            <string>全部</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>职工</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>居民</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>科别·医生</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>全部</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>申请日期</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>姓名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>病种</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>电话</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>身份证</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>科室</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>医生</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>交接情况</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>领卡日期</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
